cmd/coreinstance/create_core_instance_k8s.go

```diff
@@ -166,3 +182,14 @@ func newCmdCreateCoreInstanceOnK8s(config *cfg.Config, testClientSet kubernetes.
 
 	return cmd
 }
+
+// GetK8sYaml strips empty properties which would typically be marshalled with yaml.Marshal
+func GetK8sYaml(req interface{}) {
```

Comment:
```suggestion
func printK8sYaml(req any) {
```

"Get" denotes getting a value, but this function doesn't return anything. Also, it's not reused outside so why export it.